test_app/lib/screens/join_screen.dart

```diff
@@ -292,17 +331,22 @@ class _JoinConferenceContentState extends State<JoinConferenceContent> {
   ConferenceCreateOption conferenceCreateOptions() {
     var conferenceName = conferenceAliasTextController.text;
     var params = ConferenceCreateParameters();
-    params.dolbyVoice = switchDolbyVoice;
+    params.dolbyVoice = switchDolbyVoiceWithMixer ? switchDolbyVoiceWithMixer : switchDolbyVoiceWithoutMixer;
     params.liveRecording = true;
-    var createOptions = ConferenceCreateOption(conferenceName, params, 0);
+    var createOptions = ConferenceCreateOption(conferenceName, params, 0, SpatialAudioStyle.disabled);
+    if (switchSpatialAudioWithIndividual == true) {
+      createOptions.spatialAudioStyle = SpatialAudioStyle.individual;
+    } else if (switchSpatialAudioWithShared == true) {
+      createOptions.spatialAudioStyle = SpatialAudioStyle.shared;
+    }
     return createOptions;
   }
 
   ConferenceJoinOptions conferenceJoinOptions() {
     var joinOptions = ConferenceJoinOptions();
     joinOptions.constraints = ConferenceConstraints(true, true);
     joinOptions.maxVideoForwarding = 4;
-    joinOptions.spatialAudio = switchSpatialAudio;
+    joinOptions.spatialAudio = switchSpatialAudioWithShared? switchSpatialAudioWithShared : switchSpatialAudioWithIndividual;
```

Comment:
Is this not as simple `or`?

Reply:
I think you’re right, it can be made simple by "or".